src/pairs/Player.py

```diff
@@ -1,12 +1,14 @@
+import copy
+
 class Player:
     def __init__(self, original_deck, points=0, hand_state=None, hand_capacity=None):
         self.original_deck = original_deck
         self.points = points
         if hand_state is None:
-            hand_state = self.hand_state = {card_type: 0 for card_type in original_deck}
+            hand_state = self.hand_state = {card_type: 0 for card_type in original_deck.original_deck}
```

Comment:
The `original_deck.original_deck` is redundant. How about when you initialize your players, instead of passing in the `Deck` object for `original_deck`, you pass in the Python dict for the original deck?

Reply:
fixed